crates/sui-indexer-alt-jsonrpc/src/api/objects/mod.rs

```diff
@@ -65,6 +87,50 @@ pub struct ObjectsConfig {
 
 #[async_trait::async_trait]
 impl ObjectsApiServer for Objects {
+    async fn get_object(
+        &self,
+        object_id: ObjectID,
+        options: Option<SuiObjectDataOptions>,
+    ) -> RpcResult<SuiObjectResponse> {
+        let Self(ctx, _) = self;
+        let options = options.unwrap_or_default();
+        Ok(response::live_object(ctx, object_id, &options)
+            .await
+            .with_internal_context(|| {
+                format!("Failed to get object {object_id} at latest version")
+            })?)
+    }
+
+    async fn multi_get_objects(
+        &self,
+        object_ids: Vec<ObjectID>,
+        options: Option<SuiObjectDataOptions>,
+    ) -> RpcResult<Vec<SuiObjectResponse>> {
+        let Self(ctx, config) = self;
+        if object_ids.len() > config.max_multi_get_objects {
+            return Err(invalid_params(Error::TooManyKeys {
+                requested: object_ids.len(),
+                max: config.max_multi_get_objects,
+            })
+            .into());
+        }
+
+        let options = options.unwrap_or_default();
+
+        let obj_futures = object_ids
+            .iter()
+            .map(|id| response::live_object(ctx, *id, &options));
+
+        Ok(future::join_all(obj_futures)
```

Comment:
Is my understanding correct that by doing `future::join_all` here, we'll be able to group all the `load_one`s in `response::live_object` and take advantage of the data loader?

Reply:
That's correct